flatten_sctp.py

```diff
@@ -20,14 +20,17 @@
 for pkt in pcap:
     ip = pkt['IP']
     layer = ip.payload
+    sctp = ip.payload
     while layer.name != 'NoPayload':
         if layer.name == 'SCTP':
             sport = layer.sport
```

Comment:
I guess we can mostly remove this or replace this with `sctp = layer` and forget about sport/dport/tag? WDUT?